.github/workflows/python-package.yml

```
@@ -54,3 +54,9 @@ jobs:
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
         run: twine upload */*
+      - name: Upload release artifact
+        uses: softprops/action-gh-release@v0.1.14
+        with:
+          files: |
+            *.tar.gz
+            *.whl
```

For whatever reason this doesn't work (testing on a fork). [I got it to pick up the files](https://github.com/austinmatherne-wk/Arelle/releases/tag/2.2.7) by using the same pattern as the twine command: 
```suggestion
          files: '*/*'
```

Can you also set [`fail_on_unmatched_files`](https://github.com/softprops/action-gh-release#inputs) to true for all of these?